requirements/requirements_nlp.txt

```diff
@@ -5,11 +5,12 @@ fasttext
 flask_restful
 ftfy
 gdown
-gradio
+gradio==3.28.3
```

Comment:
@yidong72 Can this be made >= please? Its interfering with some other projects.

Reply:
yes. I will change it